src/api/fromUrlStatus.ts

```diff
@@ -78,7 +79,7 @@ const getRequestQuery = (token: string, settings: Settings) => ({
   pub_key: settings.publicKey || '',
 })
 
-const getRequestOptions = (token: string, settings: Settings): RequestOptions => {
+const getRequestOptions = (token: Uuid, settings: Settings): RequestOptions => {
```

Comment:
`token: Token` ?

I suggest to use Uuid for files.

Reply:
Yes, good. I agree with you